backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc

```diff
@@ -34,6 +34,7 @@ include::modules/restore-determine-state-etcd-member.adoc[leveloffset=+1]
 Depending on the state of your unhealthy etcd member, use one of the following procedures:
 
 * xref:../../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member[Replacing an unhealthy etcd member whose machine is not running or whose node is not ready]
+* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/expanding-the-cluster#installing-primary-control-plane-node-unhealthy-cluster_expanding-the-cluster[Replacing a control plane node in an unhealthy cluster (for Assisted Installer)]
```

Comment:
This technically links to "Installing a primary control plane node on an unhealthy cluster". I'm guessing that this is intentional? I always think it's odd when we change the friendly reference text from the subheading text. Could it potentially confuse someone?

Reply:
Updated as suggested.